Sources/AppStoreConnectCLI/Services/Operations/GetUserInfoOperation.swift

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct GetUserInfoOperation: APIOperation {
+
+    struct GetUserInfoDependencies {
+        let usersResponse: (APIEndpoint<UsersResponse>) -> Deferred<Future<UsersResponse, Error>>
+    }
+
+    enum GetUserInfoError: LocalizedError {
+        case couldNotFindUser(email: String)
+
+        var failureReason: String? {
+            switch self {
+            case .couldNotFindUser(let email):
+                return "Couldn't find user with input email: '\(email)' or email not unique"
+            }
+        }
+    }
+
+    private let endpoint: APIEndpoint<UsersResponse>
+    private let email: String
+
+    init(options: GetUserInfoOptions) {
+        let filters: [ListUsers.Filter] = [.username([options.email])]
+        endpoint = APIEndpoint.users(filter: filters)
+        email = options.email
+    }
+
+    func execute(with dependencies: GetUserInfoDependencies) -> AnyPublisher<User, Error> {
+        dependencies.usersResponse(endpoint)
+            .tryMap { [email] response in
+                let users = User.fromAPIResponse(response)
```

Comment:
```suggestion
                let users = [User](response)
```

Reply:
This change won't compile as is, would have to write an array based initializer for `User` that takes a `UserResponse`